delivery_carrier_label_postlogistics/views/res_config.xml

```diff
@@ -9,146 +9,121 @@
     <field name="arch" type="xml">
       <xpath expr="//div[hasclass('settings')]" position="inside">
         <div class="app_settings_block" data-string="Postlogistics" string="Postlogistics" data-key="postlogistics">
-        <h2>Web Service Authentication</h2>
-        <div class="row mt16 o_settings_container" id="postlogistics_ws_authentication">
-          <div class="col-xs-12 col-md-6 o_setting_box" id="credentials">
-            <div class="o_setting_right_pane">
-              <span>Credentials</span>
-              <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." groups="base.group_multi_company"/>
-              <div class="text-muted">
-                For "Swiss Post Business customer" account
-              </div>
-              <div class="content-group">
-                <div class="row mt16">
-                  <label class="col-md-3 o_light_label" for="username"/>
-                  <field name="username" />
-                </div>
-                <div class="row">
-                  <label class="col-md-3 o_light_label" for="password" />
-                  <field name="password" password="1" />
-                </div>
+          <h2>Post digital commerce API Authentication</h2>
+          <div class="row mt16 o_settings_container" id="postlogistics_ws_authentication">
+            <div class="col-xs-12 col-md-6 o_setting_box" id="credentials">
+              <div class="o_setting_right_pane">
+                <button name='%(delivery_carrier_label_postlogistics.action_postlogistics_auth)d' type="action"
+                        string="Oauth Authenticate" class="oe_link"/>
               </div>
             </div>
           </div>
-        </div>
```

Comment:
why remove this div ?

Reply:
Because now I just have a button to open wizard.
Then the previous div for user and password is useless.